recipes/R/Retail - GradientBoosting/R/applicationTrainer.R

```diff
@@ -39,59 +39,12 @@ applicationTrainer <- setRefClass("applicationTrainer",
       library(tidyverse)
       set.seed(1234)
```

Comment:
Curious - in the applicationEvaluator.R file we read the `seed` value from the config.  Here, we don't.  Is that optimal?

Reply:
Yes. In applicationEvaluator.R we are using evaluation.seed which is only for evaluation.